versioned_docs/version-0.x.0/kubeslice-cli/installing-kubeslice.mdx

```diff
@@ -0,0 +1,234 @@
+# Installing KubeSlice
+
+There are the five main steps for installing the KubeSlice:
+
+- Installing the KubeSlice Controller
+- Creating a project
+- Registering the worker clusters
+- Creating a slice
+- Onboarding namespaces onto the slices
+
+This topic describes the steps to install KubeSlice in your existing multi-cluster configuration.
+
+## Installing KubeSlice in a Multi-Cluster Configuration
+
+Kubeslice-cli is a tool that makes it simple to install Kubeslice into your existing clusters. The primary purpose of the tool is to install 
+kubeslice into your already existing clusters using the description of those clusters that you provide in a **custom topology** YAML file.
+Depending on the requirements, the Kubeslice Controller and/or worker components can be installed either incrementally (one cluster at a time) 
+or across all the clusters in your topology.
+
+For demonstration purposes, to set up a three-cluster topology of kind clusters in your local environment, use the 
+**—profile <minimal-demo | full-demo>** option instead of the **—config** option.
+
+## Installing the KubeSlice Controller and Worker Clusters
+
+You must create a topology configuration file that includes the names of the clusters, the project name, and the cluster contexts
+that host the KubeSlice Controller and the worker clusters. For more information, see sample 
+[topology configuration](/versioned_docs/version-0.x.0/kubeslice-cli/reference/sample-configuration-file.mdx) file.
+
+Use the following command to install the controller and worker cluster:
+```
+kubeslice-cli --config=<topology-configuration-file> install
+```
+The above command installs the KubeSlice Controller, creates a project, and registers the worker cluster with the project by installing the Slice
+Operator on the worker cluster.
+
+## Registering a Worker Cluster
+
+The kubeslice-cli allows you to add a new worker cluster to an existing KubeSlice configuration.
+
+Use the following template to add a new worker cluster.
+
+```
+configuration:
+  cluster_configuration:
+    kube_config_path: <kubeconfig-file>
+    controller:
+      name: <controller-cluster-name>
+      context_name: <controller-cluster-context>
+    workers:
+      - name: <new-worker-cluster-name>
+        context_name: <new-worker-cluster-context>
+  kubeslice_configuration:
+    project_name: <project-namespace>
+```
+The following is an example topology file for registering a new worker cluster.
+```
+configuration:
+  cluster_configuration:
+    kube_config_path: <PATH-TO-KUBECONFIG>
+    controller:
+      name: controller
+      context_name: kind-controller
+    workers:
+      - name: worker-3
+        context_name: ks-w-3      
+  kubeslice_configuration:
+    project_name: demo
+  ```
+
+:::note
+If the KubeSlice Controller is already installed, the `-s controller` option skips installing it. The `-s kind` option skips creating a new kind 
```

Comment:
when you use `a`, the following object should be a singular object

Here `creating a new kind cluster`..

Reply:
updated